### DISCUSSION TITLE
Pynecone Discord

### DISCUSSION THREAD

**Discussion by Alek99** [reflex-dev/reflex#233] [Announcements]
Hi everyone we just started using discord yesterday. It's a great way to get more realtime answers and feedback on Pynecone. We can help you debug your Pynecone app and collaborate and features/bug fixes! 

- We will still use GitHub issues and discussions this is just a supplement to get more specialized help and collaboration.

Invite to Pynecone Discord: https://discord.gg/T5WSbC2YtQ

**Comment by AlexanderMaxRanabel** [comment]
I Think you should open a matrix space too
